src/settings-ui/Settings.UI/Controls/SettingsGroup/SettingsGroup.xaml

```diff
@@ -8,6 +8,7 @@
             <Setter.Value>
                 <ItemsPanelTemplate>
                     <StackPanel
+                        ChildrenTransitions="{StaticResource SettingsCardsAnimations}"
```

Comment:
I read that

>If you set an EntranceThemeTransition animation on a panel, the children of the panel will automatically offset when they animate into view to create a visually appealing entrance.

So, is it needed to use ChildrenTransitions? Or: would it be possible to set the animation/style even higher up in the hierarchy, like the ScrollViewer or SettingsPageControl?

Reply:
Since we have StackPanels in StackPanels, we'd need to set that into each one individually - it doesn't trickle down.